src/hw-4/models/group.model.ts

```diff
@@ -0,0 +1,23 @@
+import { DataTypes } from 'sequelize';
+
+export default (sequelize: any) => {
+    return sequelize.define('group', {
+        id: {
+            type: DataTypes.UUID,
+            defaultValue: DataTypes.UUIDV4,
+            primaryKey: true,
+            allowNull: false
+        },
+        name: {
+            type: DataTypes.TEXT,
+            allowNull: false
+        },
+        permission: {
+            type: DataTypes.ARRAY(DataTypes.TEXT),
+            allowNull: false
+        }
+    }, {
+        timestamps: false,
+        tableName: 'group'
```

Comment:
`Groups` according to the hometask description. I think it's not crucial here, but if there was several apps working with the table or several people working on the different layers such contract would be important.